nodejs/esm.mjs

```diff
@@ -0,0 +1,115 @@
+import newrelic from 'newrelic'
+import fs from 'node:fs'
+import path from 'node:path'
+
+process.env.NEW_RELIC_APP_NAME = process.env.NEW_RELIC_APP_NAME || process.env.AWS_LAMBDA_FUNCTION_NAME
+process.env.NEW_RELIC_DISTRIBUTED_TRACING_ENABLED = process.env.NEW_RELIC_DISTRIBUTED_TRACING_ENABLED || 'true'
```

Comment:
DT is now enabled by default, so we can likely dispense with this in both wrappers.

Reply:
sure but the scope of this Pr was to work with response streaming. the serverless team can PR that if they want to in the future